packages/commonjs/types/index.d.ts

```diff
@@ -109,17 +109,17 @@ interface RollupCommonJSOptions {
    * they should be left unconverted as it requires an optional dependency
    * that may or may not be installed beside the rolled up package.
    * Due to the conversion of `require` to a static `import` - the call is
-   * hoisted to the top of the file, outside of the `try-catch` clause.
+   * hoisted to the top of the file, outside the `try-catch` clause.
```

Comment:
not really a typo but, sure

Reply:
This is picked up by Webstorm's LanguageTool and as such was underlined as a typo